charts/firefly-perfnode/values.yaml

```diff
@@ -0,0 +1,156 @@
+# Copyright © 2022 Kaleido, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://swww.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+nameOverride: ""
+fullnameOverride: ""
+
+firefly:
+  fullnameOverride: ""
+
+  # only set this after the first perfnode has been deployed
+  # use `kubectl exec ... -- /bin/sh -c 'ipfs id' | jq '.Addresses'`
+  ipfs:
+    bootstrap: ""
+
+  config:
+    ipfsApiUrl: http://127.0.0.1:5001
+    ipfsGatewayUrl: http://127.0.0.1:8080
+    postgresUrl: postgres://postgres:f1refly@127.0.0.1:5432?sslmode=disable
+    postgresAutomigrate: true
+
+  core:
+    jobs:
+      postgresMigrations:
+        enabled: false
+      registration:
+        enabled: true
+
+    service:
+      httpPort: 3000
+      adminPort: 3001
+      debugPort: 3030
+      metricsPort: 3100
+
+    initContainers: ""
+
+    # Because the swarm key is hardcoded, multiple firefly _networks_ installed on the same k8s cluster via
```

Comment:
Are the perf tests currently running with multiple FF networks on a single cluster?

Reply:
Currently no, and bc multicast doesn't work on K8s by default this is not much of an issue. IPFS nodes have to be told at least one other member's IP in order to cluster, so two clusters accidentally joining each other isn't really possible w/o manual intervention.